tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -6,11 +6,7 @@
 from typing import Any, ClassVar
 
 from typing import overload
-import pandas
-# stubgen override - unfortunately we need this for version checks
-import sys
-# stubgen override - This should probably not be exposed
-#_clean_default_connection: Any
+_clean_default_connection: PyCapsule
```

Comment:
The stubgen  makes mistakes here, so whenever ```# stubgen override``` is present it means these were manual changes that must be kept even if the script changes it, my bad for not letting you know before